ew.py

```diff
@@ -227,10 +255,23 @@ def die(self, cause = None):
 		ewutils.moves_active[self.id_user] = 0
 		ewstats.clear_on_death(id_server = self.id_server, id_user = self.id_user)
 
+		self.persist()
+
+		if cause not in explosion_block_list: # Run explosion after location/stat reset, to prevent looping onto self
+			if user_hasCombustion:
+				explode_resp = "\n{} spontaneously combusts, horribly dying in a fiery explosion of slime and shrapnel!! Oh, the humanity!\n".format(server.get_member(self.id_user).display_name)
+				ewutils.logMsg("")
+				resp_cont.add_channel_response(explode_poi_channel, explode_resp)
+
+				explosion = ewutils.explode(damage = explode_damage, district_data = explode_district)
```

Comment:
we need to remove the explode() call from explode()

Reply:
I agree, forgot to redo that change after i accidentally lost the whole project